sklearn/utils/estimator_checks.py

```diff
@@ -3187,3 +3188,29 @@ def check_estimator_get_tags_default_keys(name, estimator_orig):
         f"{name}._get_tags() is missing entries for the following default tags"
         f": {default_tags_keys - tags_keys.intersection(default_tags_keys)}"
     )
+
+
+def check_meta_estimators_validation(name, estimator_orig):
+    # Check that meta-estimators delegate data validation to the inner
+    # estimator(s).
+    rng = np.random.RandomState(0)
+
+    estimator = clone(estimator_orig)
+    set_random_state(estimator)
+
+    X, y = fetch_20newsgroups(return_X_y=True, random_state=rng)
```

Comment:
For this test, I think should have a simple text generation process and not use the network. Maybe:

```py
rng = np.random.RandomState(42)
X = rng.choice(np.array(["a", "b", "c"], dtype=object), size=30)
y = rng.randint(2, size=30)
```

Reply:
good point. Done